pyvrp/cpp/ProblemData.h

```diff
@@ -348,6 +353,12 @@ class ProblemData
      *     Maximum route duration. Unconstrained if not explicitly provided.
      * max_distance
      *     Maximum route distance. Unconstrained if not explicitly provided.
+     * unit_distance_cost
+     *     Cost per unit of distance traveled by vehicles of this type. Default
+     *     1.
+     * unit_duration_cost
+     *     Cost per unit of duration on routes serviced by vehicles of this
```

Comment:
"serviced" is a bit confusing because we also have service duration. Idem below.

```suggestion
     *     Cost per unit of duration on routes using vehicles of this
```